### DISCUSSION TITLE
How to Send Array Query Parameters using array[]=val1&array[]=val2 format in GET Requests (try it)?

### DISCUSSION THREAD

**Discussion by michaelpoernomo** [dedoc/scramble#630] [Q&A]
When using "test request" (try it) to send a GET request, is it possible to structure the query parameter for an array like this:

```
/api/test?columns[]=id&columns[]=name
```

Instead of the default format:

```
/api/test?columns=id&columns=name
```

This is because Laravel, by default, expects array query parameters in the columns[] format rather than columns.

Here are the validation rules I use in my Laravel application:

```
public function rules(): array
{
    return [
        'limit' => 'sometimes|integer',
        'keyword' => 'sometimes|string|max:255',    
        'columns' => 'sometimes|array',
    ];
}
```

<img width="449" alt="Screenshot 2024-11-20 at 14 34 13" src="https://github.com/user-attachments/assets/56dc7d78-c8e0-43a3-9f3f-7ccbcfd1f038">

**Comment by poldixd** [comment]
Did you find a solution for this?

**Reply by romalytvynenko** [reply]
@poldixd still looking for the best way to represent this case in OpenAPI document.

Please check the discussion here https://github.com/dedoc/scramble/issues/767 and share your thoughts
